Sources/Subprocess/Platforms/Subprocess+Windows.swift

```diff
@@ -473,15 +445,8 @@ internal func monitorProcessTermination(
         if let waitHandle {
             _ = UnregisterWait(waitHandle)
         }
-    }
-    guard
-        let processHandle = OpenProcess(
-            DWORD(PROCESS_QUERY_INFORMATION | SYNCHRONIZE),
-            false,
-            pid.value
-        )
-    else {
-        return .exited(1)
+
+        _ = CloseHandle(execution.processHandle)
```

Comment:
My understanding is that a CloseHandle error is _usually_ programmer error, but should this assert/throw if it does return one?